File: workspace/pset6/Sentimental/mario/mario.py
from cs50 import get_int

def main():
    height = get_int("Height: ")
    while(height < 0 or height > 23):
        height = get_int("Height: ")


    count = 1
    count2 = 1
    for j in range(height):
        for i in range(height - count):
            print(" ", end = "")

        count += 1
        for k in range(count - 1):
            print("#", end = "")

        print(" ", end = "")
        print(" ", end = "")
        for l in range(count2):
            print("#", end = "")

        count2 += 1
        #new line
        print(" ")

if __name__ == "__main__":
    main()

File: workspace/pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int height = 0;
    do{
        height = get_int("Height: ");
    }
    while(height < 0 || height > 23);


    int count = 1;
    int count2 = 1;
    for(int j = 0; j < height; j++){
        for(int i = 0; i < height - count; i++){
            printf(" ");
        }
        count++;
        for(int k = 0; k < count - 1; k++){
            printf("#");
        }
        printf(" ");
        printf(" ");
        for(int l = 0; l < count2; l++){
            printf("#");
        }
        count2++;
        printf("\n");
    }
}

